HierarchicalDataConverter/Catalyst.HierarchicalDataConverter/HierarchicalDataTransformer.cs

```diff
@@ -252,7 +252,8 @@ private void ValidateHierarchicalBinding(Binding binding, Binding[] allBindings)
             bool isFirst)
         {
             var sourceEntity = await this.GetEntityFromBinding(rootBinding);
-            LoggingHelper2.Debug($"GenerateDataSources -- sourceEntity: {JsonConvert.SerializeObject(sourceEntity)}");
+
+            // LoggingHelper2.Debug(this.guid, $"GenerateDataSources -- sourceEntity: {JsonConvert.SerializeObject(sourceEntity)}");
```

Comment:
Should we just delete these debug lines? or are we planning to restore them later?

Reply:
They will all get cleaned up today-ish when we tackle the logging pr.  Looks like we've been going back and forth deleting some and adding some back in between our prs.  Probably okay to reconcile it all together in the logging pr